legend-engine-xt-nonrelationalStore-mongodb-javaPlatformBinding-pure/src/main/resources/core_nonrelational_mongodb_java_platform_binding/test/testSetup.pure

```diff
@@ -166,7 +168,7 @@ function meta::external::store::mongodb::executionPlan::platformBinding::legendJ
             meta::pure::mapping::modelToModel::executionPlan::platformBinding::legendJava::inMemoryLegendJavaPlatformBindingExtension(),
             meta::external::shared::format::executionPlan::platformBinding::legendJava::bindingLegendJavaPlatformBindingExtension([
                meta::external::format::json::executionPlan::platformBinding::legendJava::jsonSchemaJavaBindingDescriptor()
```

Comment:
this is a duplicated function .. there is another place where we define the required extensions (where we updating the MongoOnly extensions for the server)

Reply:
i remove the whole file